.gitignore

```diff
@@ -1,4 +1,64 @@
-npm-debug.log
+package-lock.json
+
+### Node template
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
```

Comment:
no need so much ignore files :)

Reply:
This is template of Ignore plugin in JetBrains Idea ))